src/m365/teams/commands/meeting/meeting-get.spec.ts

```diff
@@ -0,0 +1,281 @@
+    const actual = await command.validate({
+      options: {
+        userId: 'foo',
+        joinUrl: joinUrl
+      }
+    }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('passes validation if required options specified', async () => {
+    const actual = await command.validate({
+      options: {
+        userId: userId,
+        joinUrl: joinUrl
+      }
+    }, commandInfo);
+    assert.strictEqual(actual, true);
+  });
+
+  it('retrieves specific meeting details using userId (debug)', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/users/${userId}/onlineMeetings?$filter=JoinWebUrl eq '${encodeURIComponent(joinUrl)}'`) {
+        return meetingResponse;
+      }
+
+      throw `Invalid request`;
+    });
+
+    await command.action(logger, {
+      options: {
+        debug: true,
+        userId: userId,
+        joinUrl: joinUrl
+      }
+    });
+
+    assert(loggerLogSpy.calledWith(meetingResponse.value[0]));
+  });
+
+  it('retrieves specific meeting details using userName', async () => {
+    sinon.stub(Cli, 'executeCommandWithOutput').callsFake(async (command): Promise<any> => {
+      if (command === userGetCommand) {
+        return { "stdout": JSON.stringify({ id: userId }) };
+      }
+      throw 'Invalid request';
+    });
+
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/users/${userId}/onlineMeetings?$filter=JoinWebUrl eq '${encodeURIComponent(joinUrl)}'`) {
+        return meetingResponse;
+      }
+
+      throw `Invalid request`;
+    });
+
+    await command.action(logger, {
+      options: {
+        debug: true,
+        userName: userName,
+        joinUrl: joinUrl
+      }
+    });
+
+    assert(loggerLogSpy.calledWith(meetingResponse.value[0]));
+  });
+
+  it('retrieves specific meeting details using email', async () => {
+    sinon.stub(Cli, 'executeCommandWithOutput').callsFake(async (command): Promise<any> => {
+      if (command === userGetCommand) {
+        return { "stdout": JSON.stringify({ id: userId }) };
+      }
+      throw 'Invalid request';
+    });
+
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/users/${userId}/onlineMeetings?$filter=JoinWebUrl eq '${encodeURIComponent(joinUrl)}'`) {
+        return meetingResponse;
+      }
+
+      throw `Invalid request`;
+    });
+
+    await command.action(logger, {
+      options: {
+        debug: true,
+        email: email,
+        joinUrl: joinUrl
+      }
+    });
+
+    assert(loggerLogSpy.calledWith(meetingResponse.value[0]));
+  });
+
+  it('correctly handles error when getting specified meeting details', async () => {
+    const errorMessage = 'request rejected';
+    sinon.stub(request, 'get').callsFake(async () => { throw errorMessage; });
```

Comment:
It's very unlikely that the API will just throw a plain error message. Instead we should throw a realistic error object.
This commands throws an error like this:

```
"error": {
    "error": {
      "message": "An error has occurred.",
    }
  }
```